sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaWriteSchemaTransformProvider.java

```diff
@@ -180,8 +211,14 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
                   .withKeySerializer(ByteArraySerializer.class)
                   .withValueSerializer(ByteArraySerializer.class));
 
+      // TODO: include output from KafkaIO Write once updated from PDone
+      PCollection<Row> errorOutput =
+          outputTuple.get(ERROR_TAG).setRowSchema(ErrorHandling.errorSchema(errorSchema));
       return PCollectionRowTuple.of(
-          "errors", outputTuple.get(ERROR_TAG).setRowSchema(ERROR_SCHEMA));
+          handleErrors
+              ? Objects.requireNonNull(configuration.getErrorHandling()).getOutput()
```

Comment:
nit: again, redundant since `ErrorHandling.hasOutput(configuration.getErrorHandling());` is called.
```suggestion
              ? configuration.getErrorHandling().getOutput()
```